mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -500,6 +500,7 @@ def execute(self):
             if self.options.skip_flashing:
                 self.logger.prn_inf("copy image onto target... SKIPPED!")
             else:
+                result = None
```

Comment:
Don't think this line is necessary since it will be replaced below by the call to `self.mbed.copy_image()`, any chance we can remove it?